cli/link_parquets.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+python3.8 -m ingest.process.entity_linking.link --input-path /input --output-path /output/ \
+    --cluster tcp://scheduluer:8786 --dataset-id documents
```

Comment:
Spelling on scheduluer

Reply:
It actually was a big deal ;)  Fixed in 0463b8d